mlir/test/Dialect/SCF/transform-ops.mlir

```diff
@@ -270,3 +270,32 @@ module attributes {transform.with_named_sequence} {
     transform.yield
   }
 }
+
+// -----
+
+// CHECK-LABEL: func @coalesce_i32_loops(
+
+func.func @coalesce_i32_loops() {
```

Comment:
Please, add comment to the test what is actually being tested (i.e. coalese failure with non-index types).

Reply:
Done